components/camera/videosource/webcam.go

```diff
@@ -110,7 +153,8 @@ func makeConstraints(conf *WebcamConfig, logger logging.Logger) mediadevices.Med
 			if conf.FrameRate > 0.0 {
 				constraint.FrameRate = prop.FloatExact(conf.FrameRate)
 			} else {
-				constraint.FrameRate = prop.FloatRanged{Min: 0.0, Ideal: 30.0, Max: 140.0}
+				conf.FrameRate = 30.0
```

Comment:
Is this supposed to use the new `defaultFrameRate`?

...why did this line change at all? Looks like I made a comment about this last time, too, and you marked it as resolved but I've forgotten why that happened. What was wrong with the old approach? Put a comment here in Github.